db/migrate/20230126025440_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :AuthorId
```

Comment:
- [ ] OPTIONAL
Kindly note that we don't have to explicitly declare `AuthorId`. This should be added automatically to the schema (`author_id`) on the associations related to the `user` model is properly established.